.github/actions/k-bench/README.md

```diff
@@ -0,0 +1,157 @@
+
+Example table:
+
+<details>
+
+| Benchmark suite | Current | Previous | Ratio |
+|-|-|-|-|
+| pod_create (ms) | 306.521 | 196.762 | 1.558 ⬆️ |
+| pod_list (ms) | 157.272 | 2083.966 | 0.075 ⬇️ |
+| pod_get (ms) | 157.42899 | 99.745 | 1.578 ⬆️ |
+| pod_update (ms) | 176.695 | 150.139 | 1.177 ⬆️ |
+| pod_delete (ms) | 165.996 | 449.71698 | 0.369 ⬇️ |
+| svc_create (ms) | 208.479 | 407.957 | 0.511 ⬇️ |
+| svc_list (ms) | 154.088 | 1398.8309 | 0.11 ⬇️ |
+| svc_get (ms) | 156.53699 | 100.264 | 1.561 ⬆️ |
+| svc_update (ms) | 157.96 | 127.22 | 1.242 ⬆️ |
+| svc_delete (ms) | 193.49901 | 914.974 | 0.211 ⬇️ |
+| depl_create (ms) | 347.759 | 212.902 | 1.633 ⬆️ |
+| depl_list (ms) | 158.06 | 2060.005 | 0.077 ⬇️ |
+| depl_update (ms) | 175.196 | 124.198 | 1.411 ⬆️ |
+| depl_scale (ms) | 161.622 | 215.881 | 0.749 ⬇️ |
+| depl_delete (ms) | 157.02899 | 178.04 | 0.882 ⬇️ |
+| net_internode_snd (Mbit/s) | 979.0 | 1090.0 | 1.113 ⬆️ |
+| net_intranode_snd (Mbit/s) | 20700.0 | 30000.0 | 1.449 ⬆️ |
+| fio_root_async_R70W30_R (MiB/s) | 0.4541015625 | 5.1953125 | 11.441 ⬆️ |
+| fio_root_async_R70W30_W (MiB/s) | 0.2001953125 | 2.2353515625 | 11.166 ⬆️ |
+| fio_root_async_R100W0_R (MiB/s) | 0.58984375 | 39.0 | 66.119 ⬆️ |
+| fio_root_async_R0W100_W (MiB/s) | 1.1806640625 | 2.3896484375 | 2.024 ⬆️ |
+
+</details>
+
+### Drawing Performance Charts
+The action also draws graphs as used in the [Constellation docs](https://docs.edgeless.systems/constellation/next/overview/performance). The graphs compare the performance of Constellation to the performance of managed Kubernetes clusters.
+
+Graphs are created with every run of the benchmarking action. The action attaches them to the `benchmark` artifact of the workflow run.
+
+## Updating Stored Records
+
+### Managed Kubernetes
+One must manually update the stored benchmark records of managed Kubernetes:
+
+### AKS
+Follow the [Azure documentation](https://learn.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-portal?tabs=azure-cli) to create an AKS cluster of desired benchmarking settings (region, instance types). If comparing against Constellation clusters with CVM instances, make sure to select the matching CVM instance type on Azure as well.
+
+Once the cluster is ready, set up managing access via `kubectl` and take the benchmark:
+```bash
+# Setup
+git clone https://github.com/edgelesssys/k-bench.git
+cd k-bench && git checkout feat/constellation
+./install.sh
+
+# Remove the Constellation encrypted storage class 
+# Remember to revert this change before running K-Bench on Constellation!
+yq 'del(.spec.storageClassName)' config/dp_fio/fio_pvc.yaml
+yq 'del(.spec.storageClassName)' config/dp_netperf_internode/netperf_pvc.yml
+yq 'del(.spec.storageClassName)' config/dp_network_internode/netperf_pvc.yaml
+yq 'del(.spec.storageClassName)' config/dp_network_intranode/netperf_pvc.yml
```

Comment:
You could generally avoid this by assuming that "encrypted-storage" is also the default storage class on Constellation. I think if you follow the docs it actually should be the default one.

Reply:
You're right, but I won't address it in this PR anymore.